types.go

```diff
@@ -2,6 +2,7 @@ package rsync
 
 import "github.com/gokrazy/rsync/internal/rsyncwire"
 
+// rsync/rsync.h:struct sum_buf
```

Comment:
I added this here, because it was in the removed struct's comment.

I can remove it if you don't think it has enough value.

Reply:
This can stay